cli/src/plugins/javadoc/index.ts

```diff
@@ -275,6 +275,22 @@ function makeClassDocPageBody(args: MakeSectionArgs): Node[] {
         ),
     }),
 
+    ...makeSection({
+      ...args,
+      depth,
+      title: "Enum Constant Summary",
+      shouldMakeSection: () => doc.enumConstants.length !== 0,
+      makeBody: () =>
+        makeTable(
+          ["Enum Constant and Description"],
+          doc.enumConstants.map((doc) => [
+            project.linkToEntity(
+              doc.containingClass?.asString + "_" + doc.name
```

Comment:
The first parameter should be the entity canonical name (in Javadoc, fully qualified name) and the second parameter should be the friendly name. To solve your issue, `doc.name` should be the second parameter. But I'd have to look up what the first parameter should be.

What if containingClass is null? Is it ever null?

Reply:
I believe this is correct now - in any event, `doc.qualifiedName` seems to match what is currently displaying in the existing Javadoc. But I'm curious if there's a "correct" way to know what the canonical name is.